packages/crawler/src/index.ts

```diff
@@ -17,13 +16,14 @@ interface Arguments {
     simulate: boolean;
     selectors: string[];
     output: string;
+    maxUrls: number;
 }
 
 (async () => {
     dotenv.config();
 
     const args = (yargs
-        .usage('Usage: $0 --url <url> [--simulate] [--selectors <selector1 ...>] [--output]')
+        .usage('Usage: $0 --url <url> [--simulate] [--selectors <selector1 ...>] [--output] [--maxUrls]')
```

Comment:
[--maxUrls <count>]

Reply:
not following, can you expand?

Reply:
--maxUrls option needs value to pass along, can you add that to usage description